src/platform/site-wide/va-footer/components/DesktopLinks.jsx

```diff
@@ -36,7 +38,7 @@ export default function DesktopLinks({ links, visible }) {
             </button>
           </li>
         </ul>
-        <h4 className="va-footer-linkgroup-title">Contact us</h4>
+        <h4 className="va-footer-linkgroup-title">Get answers</h4>
```

Comment:
@kelsonic  These `h4` elements in the footer should be `h2` to pull them out of their current nesting pattern. Scanners aren't showing it as an error because the heading order is technically cascading correctly, but (on the homepage, at least) the nearest h3 is the "Disability" link in the benefits section, which would imply that all the footer links are nested under it.

This is the relevant WCAG guideline https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships

Reply:
Got it, so update them to `h2`? Just want to make sure I understood correctly 🙂

Reply:
correct